File: src/main/java/me/arasple/mc/trhologram/bstats/MetricsHandler.java
package me.arasple.mc.trhologram.bstats;

import io.izzel.taboolib.module.inject.TSchedule;
import me.arasple.mc.trhologram.TrHologram;
import me.arasple.mc.trhologram.hologram.HologramManager;
import org.bstats.bukkit.Metrics;

/**
 * @author Arasple
 * @url https://bstats.org/plugin/bukkit/TrHologram
 */
public class MetricsHandler {

    @TSchedule
    public static void init() {
        Metrics metrics = new Metrics(TrHologram.getPlugin(), 6387);
        metrics.addCustomChart(new Metrics.SingleLineChart("holograms", () -> HologramManager.getHolograms().size()));
    }

}

File: src/main/java/me/arasple/mc/trhologram/bstats/MetricsHandler.kt
package me.arasple.mc.trhologram.bstats

import io.izzel.taboolib.module.inject.TSchedule
import me.arasple.mc.trhologram.TrHologram
import me.arasple.mc.trhologram.hologram.HologramManager
import org.bstats.bukkit.Metrics
import java.util.concurrent.Callable

/**
 * @author Arasple
 * @url https://bstats.org/plugin/bukkit/TrHologram
 */
object MetricsHandler {

    @TSchedule
    fun init() {

        val metrics = Metrics(TrHologram.getPlugin(), 6387)
        metrics.addCustomChart(Metrics.SingleLineChart("holograms", Callable { HologramManager.getHolograms().size }))
    }

}